src/components/network/network-explorer.js

```diff
@@ -167,6 +177,7 @@ const NetworkExplorer = ({
         network,
         identifiedElementComparator,
         generateFilteredSubstation,
+        substationsLoaded,
```

Comment:
add && substationsLoaded to the if above ?

Reply:
done